.gitignore

```diff
@@ -1,3 +1,7 @@
 .idea
 
-/tmp
\ No newline at end of file
+/tmp
+
+/utils/__pycache__
+
+/exploratory_data_analysis/outputs
```

Comment:
the new config of the yaml file doesn't need that just leave there .idea and /tmp

Reply:
this was necessary because of your previous comment that is now resolved (